Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1406,6 +1406,116 @@ function Get-CurrentUserDomain
     return $currentUserDomain
 }
 
+function Get-WebDeployParameters{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $discoveryConfig,
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $commonConfig,
+        [Parameter(Mandatory=$true)]
+        [string] $userName,
+        [string] $registrationApiSecret
+    )
+
+    Confirm-DiscoveryConfig -discoveryConfig $discoveryConfig -commonConfig $commonConfig
```

Comment:
If Confirm-DiscoveryConfig in the Discovery-Utilities module?

Reply:
Forgot to move this over